ts/index.ts

```diff
@@ -84,10 +84,22 @@ interface GossipOptions extends GossipInputOptions {
   scoreThresholds: PeerScoreThresholds
 }
 
+const ACCEPT_REQUEST_WHITE_LIST_THRESHOLD_SCORE = 0
```

Comment:
Can you move these constants to ts/constants and also add jsdoc comments for each?